dizoo/mujoco/config/halfcheetah_bdq_config.py

```diff
@@ -0,0 +1,71 @@
+from easydict import EasyDict
+
+halfcheetah_bdq_config = dict(
+    exp_name='halfcheetah_bdq_seed0',
+    env=dict(
+        env_id='HalfCheetah-v3',
+        norm_obs=dict(use_norm=False, ),
```

Comment:
remove this two lines, they are unused now

Reply:
If the 'norm_reward' parameter is not declared, the following error is displayed.

  File "/DI-engine/dizoo/mujoco/envs/mujoco_env.py", line 154, in create_evaluator_env_cfg
    evaluator_cfg.norm_reward.use_norm = False
AttributeError: 'EasyDict' object has no attribute 'norm_reward'